R/register.R

```diff
@@ -258,12 +260,33 @@ get_call_entries <- function(path) {
   close(con)
 
   start <- grep("/* .Call calls */", res, fixed = TRUE)
+
   end <- grep("};", res, fixed = TRUE)
 
   if (length(start) == 0) {
     return("")
   }
-  res[seq(start, end)]
+
+  funs$package <- package
```

Comment:
Do you need this? As far as I can tell you never use `funs$package` after this assignment.

Reply:
I used it in the next line to get the `extern SEXP _{package}_{name}` from a single tibble but I ended up sending funs$name directly and used `glue::glue()` for variables instead of `glue::glue_data`